src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -87,6 +101,11 @@ export const IncidentMap = () => {
           />
         )}
       </StyledMap>
+      <FilterPanel
+        filters={filters}
+        setFilters={setFilters}
+        setMapMessage={setMapMessage}
+      />
```

Comment:
Wat is de reden dat je deze FilterPanel hebt verplaatst?

Reply:
As discussed I will pay attention to changing code pertaining only to the PR. For now this one can remain as is. (will be changed in different story this sprint)